scripts/umbrel-os/status-server/static/index.html

```diff
@@ -67,28 +67,28 @@
       </svg>
 
       <div class="error-text monitor-check">
-        <p><b class="text-dark">Error: External drive not connected</b></p>
-        <p>This can sometimes be caused by using an unofficial power supply.</p>
+        <p><b class="text-dark">Error: External drive disconnected</b></p>
+        <p>The external drive was disconnected while Umbrel was running. This can sometimes happen when using an unofficial Raspberry Pi power supply.</p>
       </div>
 
       <div class="error-text semver-mismatch">
-        <p><b class="text-dark">Error: Can't upgrade from SDcard due to version mismatch</b></p>
-        <p>Can't upgrade from SDcard due to version mismatch.</p>
+        <p><b class="text-dark">Error: Version mismatch</b></p>
+        <p>The version of UmbrelOS on your microSD card is different than the version of Umbrel on your external drive.</p>
       </div>
 
       <div class="error-text no-block-device">
-        <p><b class="text-dark">Error: Umbrel couldn't find a storage device attached</b></p>
-        <p>Please make sure your device is attached and then reboot.</p>
+        <p><b class="text-dark">Error: No external drive found</b></p>
+        <p>Please connect an external drive (either SSD or HDD) of at least 1TB size to a USB 3.0 port (blue color) on your Raspberry Pi and restart your Umbrel.</p>
       </div>
 
       <div class="error-text multiple-block-devices">
-        <p><b class="text-dark">Error: Umbrel found multiple storage devices attached</b></p>
-        <p>Please remove one of them and reboot.</p>
+        <p><b class="text-dark">Error: Multiple external drives found</b></p>
+        <p>Umbrel only works with one external drive of atleast 1TB size. Please disconnect any additional external drives and restart your Umbrel.</p>
       </div>
 
       <div class="error-text rebinding-failed">
-        <p><b class="text-dark">Error: Umbrel failed to use your storage device in low power mode</b></p>
-        <p>Are you using the recommended hardware?</p>
+        <p><b class="text-dark">Error: Failed to rebind external drive</b></p>
```

Comment:
Maybe we can reword from "Failed to rebind external drive" to "Failed to load storage device drivers" or something more simple. I don't think many users will understand what rebinding means or that it relates to drivers.